article/sections/_7_latent_confounding.tex

```diff
@@ -0,0 +1,209 @@
+	variables to recover the substitute confounder, our goal is to analyze 
+	whether we can improve the accuracy of this approach by adopting a stronger 
+	prior on which variables are confounded. This can 
+	be in the form of building and testing candidate causal graphs that 
+	try to illustrate this confounding. 
+\end{itemize}
+
+
+We compare both the coefficient estimates on travel time and cost from 
+each of those three models to the true estimates used in the simulation. 
+We also compare the distribution of the recovered substitute confounder under each of 
+models 2 and 3 to the true confounder. The main findings of this exercise are the following:
+
+
+Using the true variables believed to be confounded (i.e. method 3 where only 
+travel time and cost are used to recover the confounder) leads to a better 
+recovery of the true confounder. Figure \ref{fig:qq-plot-method-2} and \ref{fig:qq-plot-method-3} show a QQ plot of the true 
+and recovered confounders under models 2 and 3 respectively. Looking at those 
+figures, we see that the distribution of the recovered substitute 
+confounder under method 3 is closer to that of the true confounder than method 
+2. This suggests that it may be better to run the deconfounder algorithm based 
+on a hypothesized causal graph, rather than just running it on all the 
+observed covariates. Please refer to Section \ref{sec:graph-construction} for how to build plausible causal graphs. 
+
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-2.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
+   \label{fig:qq-plot-method-2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-3.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
+   \label{fig:qq-plot-method-3}
+\end{figure}
+
+% \begin{figure}
+% \centering
+% \begin{minipage}{.5\textwidth}
+%   \centering
+%   \includegraphics[width=.4\linewidth]{qq-plot-method-2.png}
+%   \captionof{figure}{QQ plot of true confounder against recovered confounder using all covariates}
+%   \label{fig:qq-plot-method-2}
+% \end{minipage}%
+% \begin{minipage}{.5\textwidth}
+%   \centering
+%   \includegraphics[width=.4\linewidth]{qq-plot-method-3.png}
+%   \captionof{figure}{QQ plot of true confounder against recovered confounder using only confounded covariates}
+%   \label{fig:qq-plot-method-3}
+% \end{minipage}
+% \end{figure}
+
+
+Additionally, and perhaps most importantly, the effectiveness of the 
+deconfounder algorithm is very sensitive to small errors and misfits in the 
+recovered confounder. Although method 3 returns a relatively good fit of the 
+true confounder (based on the QQ plot), the adjusted coefficients on travel 
+time and cost do not exhibit any reduction in the bias resulting from 
+ommitting the true confounder. Moreover, the coefficients on the recovered 
```

Comment:
We should clarify this sentence by referring to the magnitude of the bias from method 1 for comparison. It's good to know both absolute performance (all method sucked) and relative performance (method 3 sucked, but much less than method 1 or the same?).

Reply:
Also, we should explicitly note that our factor model passed the check described in the article by Wang and Blei.

Reply:
See here for the issue to do the factor model check of Wang and Blei: #181.

Reply:
See edits in #177 for talk of the bias in coefficient recovery.